packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -42,6 +49,12 @@ const propTypes = {
    * Full Name of the person
    */
   personName: PropTypes.string,
+  /**
+   * Sets the heading level of the person's name. One of `1`, `2`, `3`, `4`, `5`, `6`.
+   * This helps screen readers announce appropriate heading levels.
+   * Changing 'personNameLevel' will not visually change the style of the content.
+   */
+  personNameLevel: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
nit: personNameHeadingLevel might be more descriptive. It's covered by the comment but could be ambiguous in code where the component is consumed.

Reply:
Updated personNameLevel prop to personNameHeadingLevel here https://github.com/cerner/terra-core/pull/3787/commits/a6720f14168f9923339af29d8db9b5b431f0598c#diff-c800d42d0a8711ffc8a30d1494c5fa290b424dedb52c4567ecfd7b2ffa9cf793R57